src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,6 @@
+@import "../utils/variables";
+
+@function gridSize($grid-item-width, $grid-num-of-item, $grid-margin-btw-item) {
```

Comment:
let's avoid using work `grid` if we don't use grid itself, it's confusing
it's OK for now, but next time — don't do it if no reason